packages/buildbucket-dart/lib/buildbucket_proto.dart

```diff
@@ -0,0 +1,8 @@
+// Copyright 2020 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+library buildbucket_proto;
```

Comment:
The standard naming convention for protos in Dart is to use pb, so `buildbucket_pb`

Reply:
done!